index.js

```diff
@@ -40,7 +40,7 @@ function Browserify (files, opts) {
     var self = this;
     if (!(this instanceof Browserify)) return new Browserify(files, opts);
     if (!opts) opts = {};
-    
```

Comment:
please revert all these unrelated whitespace changes.

Reply:
Sublime automatically cleans up trailing whitespaces. Not sure I understand what is the meaning of keeping them.
Bah, sad to see there is no way for my contributions.

Reply:
i wouldn't keep them in most situations, but browserify is old and most of the code was written by people who no longer maintain it, so history tools like `git blame` are important for people like me who have no idea why things are the way they are a lot of the time :sweat_smile: minimal patches are valuable in that respect.

thanks for the test, i hope we can get the browser-resolve patch in soon.